.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+## Description
+Please provide a short summary of the changes and the motivation behind them.
+
+## Checklist
```

Comment:
I think it would be good to add a `---` line before the `## Checklist` header. This provides a visual gap between the PR description and the PR metadata.